app/lib/geordi-logger.coffee

```diff
@@ -31,7 +40,10 @@ class GeordiLogger # Make calls to the Geordi API to log user activity
     newEntry = Object.assign {}, logEntry, @keys
     if GeordiLogger.tokens.indexOf(newEntry.projectToken) > -1
       @instance().logEvent newEntry
-      console.warn 'No Geordi logger available for event ', JSON.stringify(logEntry) unless @instance().logEvent
+      .catch (err) =>
+        if !@suppressWarnings
+          console.warn "Warning: Error was encountered logging to Geordi server [#{@state?.env} - #{@instance().GEORDI_SERVER_URL[@state?.env]}]:",err
```

Comment:
Yeah it is passed into the constructor. I know what you mean, but technically it's not reaching into another object though, only the logger itself. Anyway that's not really part of this PR, @state was already there before this PR, I just use it.